subsys/fs/littlefs_fs.c

```diff
@@ -37,18 +37,12 @@ static K_MEM_SLAB_DEFINE(file_data_pool, sizeof(struct lfs_file_data),
 static K_MEM_SLAB_DEFINE(lfs_dir_pool, sizeof(struct lfs_dir),
 			 CONFIG_FS_LITTLEFS_NUM_DIRS, 4);
 
-/* Inferred overhead, in bytes, for each k_heap_aligned allocation for
- * the filecache heap.  This relates to the CHUNK_UNIT parameter in
- * the heap implementation, but that value is not visible outside the
- * kernel.
- */
-#define FC_HEAP_PER_ALLOC_OVERHEAD 24U
```

Comment:
Leave this part, just add code here that will be setting `FC_HEAP_PER_ALLOC_OVERHEAD` to the `CONFIG_FS_LITTLEFS_HEAP_PER_ALLOC_OVERHEAD_SIZE`.
Change the description to have some TODO, FIXME to notify that this thing should be rather taken from the Kernel internals than set by user, but we do not have a way to do so now.